core/src/driver/scheduler.rs

```diff
@@ -100,7 +100,7 @@ impl SchedulerKind {
         config: AuctionTimingConfiguration,
     ) -> Box<dyn Scheduler> {
         match self {
-            SchedulerKind::System => Box::new(SystemScheduler::new(driver, config)),
+            SchedulerKind::System => Box::new(SystemScheduler::new(driver, exchange, config)),
             SchedulerKind::Evm => Box::new(EvmScheduler::new(exchange, driver, config)),
```

Comment:
uber-nano-nit: `driver` and `exchange` parameters are in switched orders for the two schedulers. Personally I think it would be nice if they were the same.